aten/src/ATen/mps/MPSFallback.mm

```diff
@@ -85,8 +85,6 @@ static Tensor slow_conv2d_forward_mps(const Tensor& self,
   // These ops are not supported via MPS backend currently, and we fallback to run on CPU.
   // For the rest of unsupported ops the user needs to pass 'PYTORCH_ENABLE_MPS_FALLBACK=1'
   // to fallback on CPU, otherwise we will error out.
-  m.impl("bitwise_left_shift.Tensor_out", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
-  m.impl("bitwise_right_shift.Tensor_out", torch::CppFunction::makeFromBoxedFunction<&mps_fallback>());
```

Comment:
If there was a fallback in place, why were things not working before?
Granted that the error was with `__left__.Scalar`. Not sure how that magic works

Reply:
Oh, the fallback was for the Tensor variant, not for the Scalar ones

Reply:
Yeah, this only works if one calls `torch.bitwise_left_shift`, not sure what's the story with all those